django/core/management/commands/dumpfixturedata.py

```diff
@@ -0,0 +1,91 @@
+from django.core.management.commands.dumpdata import Command as DumpCommand
+from django.core.management import call_command
+from StringIO import StringIO
+from django.conf import settings
+from random import randint
+from django.db.migrations.loader import MigrationLoader
+from django.db import connection
+from django.db.migrations.writer import MIGRATION_TEMPLATE
+from django import get_version
+from django.utils.timezone import now
+import os
+
+
+class Command(DumpCommand):
+
+    def handle(self, *app_labels, **options):
+
+        out = StringIO()
+        options['stdout'] = out
+        call_command("dumpdata", *app_labels, **options)
+        out.seek(0)
+        output = out.read()
+
+        json_file = "data"+str(randint(999, 9999))+".json"
+        main_app = app_labels[0].split(".")[0]
+        migrations_path = os.path.join(settings.BASE_DIR, main_app, "migrations")
+        fixture_path = os.path.join(migrations_path, "fixtures")
+        json_path = os.path.join(fixture_path, json_file)
+
+        if not os.path.isdir(fixture_path):
+            os.mkdir(fixture_path)
+
+        with open(json_path, "w") as f:
+            f.write(output)
+
+        migration_loader = MigrationLoader(connection)
+        unique_apps = set([app.split('.')[0] for app in app_labels])
+        latest_migrations_from_all_apps = [str(list(migration_loader.graph.leaf_nodes(app))[0])+"," for app in unique_apps]
+
+        call_command('makemigrations', main_app, '--empty')
```

Comment:
This creates a blank migration with a pyc file which needs to be deleted before the migration can run. Should consider writing directly to file so the pyc file does not need to be deleted. I would need to grab the naming convention from the makemigrations command.